### ISSUE TITLE
Support for Mojave?

### ISSUE THREAD

**Issue by thomaslazar** [oblador/hush#15]
Is support for older versions of MacOS possible? Currently running Mojave still so I can keep 32bit support.

**Comment by MickNotMike** [comment]
I would love this on High Sierra too, it works so well on my iOS devices and would like to use it everywhere...

**Comment by briankendall** [comment]
I tried compiling it targeting earlier versions of macOS, but as expected, it makes extensive use of APIs that are only available in Catalina or Big Sur. That said, it's only the app itself that uses these APIs, not the extension. And the app doesn't do anything other displaying whether the extension is loaded and providing some links. That could be reimplemented in an API available in High Sierra or Mojave pretty easily if someone wanted to take up the task.

**Comment by briankendall** [comment]
Alternatively, people on Mojave and earlier can install a compatible ad-blocker like AdGuard and make sure the "EasyList Cookie List" filter is enabled. That will be more or less the same thing.

**Comment by gingerbeardman** [comment]
See the fork in https://github.com/oblador/hush/pull/27

You'll need to build it yourself, but it's trivially easy.

**Comment by hontz1** [comment]
yes i did build, but easier would be auto update via appstore, also i can't sign the app because i don't have developer rights, that makes the plugin to re-aktivate after every reboot. thanks
